### PR TITLE
Add more templates to page detail

### PR DESCRIPTION
**Pull request by helloimela**
**Describe pull-request**  
- Add multiple columns
- Add spacing for images and lead text

**Solving issue**  
_Add which issue this pull-request solves by adding # plus the number of the issue (for example #123)_
Fixes: #111

### CODE REVIEW

**Comment by niklasros** [comment]
The intro blocks height varies on Level 1 and Level 2 pages. This height should be fixed, no matter if the intro block has 1 or 2 lines of text (192px).

intro-block-level1
intro-block-level2

**Comment by niklasros** [comment]
The anchor links width is too small. 

anchor-links

**Comment by helloimela** [comment]
> The anchor links width is too small.
> 
> anchor-links

@niklasros What is the screen size when you take this screenshot?

**Comment by niklasros** [comment]
> > The anchor links width is too small.
> > anchor-links
> 
> @niklasros What is the screen size when you take this screenshot?

https://deploy-preview-121--digitaldesign.netlify.app/components/buttons

**Comment by niklasros** [comment]
The text in the intro block has the wrong size. It should be $headline-03.

intro-block-text-size-mobile

**Comment by niklasros** [comment]
The default body text should be set in $body-01. Currently it is set in 14px (which should be $body-02).
Example: https://deploy-preview-121--digitaldesign.netlify.app/resources/brand/pictograms

**Comment by niklasros** [comment]
> > > The anchor links width is too small.
> > > anchor-links
> > 
> > 
> > @niklasros What is the screen size when you take this screenshot?
> 
> https://deploy-preview-121--digitaldesign.netlify.app/components/buttons

The anchor links width could still be a bit bigger, to avoid 2 lines. Try to set a max. width of 160px.
https://deploy-preview-118--digitaldesign.netlify.app/foundation/foundation-colours

**Comment by niklasros** [comment]
The anchor links have the wrong spacing between the links. It should be 8px/2rem.

anchor-links-spacing

File: src/app/components/page/tab-content/tab-content.component.html [hunk 1]
```diff
@@ -42,33 +41,59 @@
...
+              <h4 *ngIf='section.Title' [innerHTML]='section.Title | markdown | keepHtml' [attr.id]='section.Title | generateTabUrl' class="section-title"></h4>
+              <div *ngIf='section.LeadText' class="lead-text sdds-paragraph-01">{{section.LeadText.textfield}}</div>
+              <p *ngIf='section.Text' [innerHTML]='section.Text | markdown | keepHtml'></p>
             </div>
           </ng-template>
 
           <!-- Full width Image -->
           <ng-template [ngIf]="section.__typename === 'ComponentContentPluginFullWidthImage'">
-            <div class="sdds-col-xlg-16 sdds-col-md-8 sdds-col-sm-4">
+            <div class="sdds-col-xlg-16 sdds-col-md-8 sdds-col-sm-4 ful-width-image">
```
Comment (mmexvr) [inline src/app/components/page/tab-content/tab-content.component.html:53]: correct class added - full-width-image

**Comment by niklasros** [comment]
The images along with the image captions are not aligned with the grid. 

detailed-page-image-alignment

**Comment by niklasros** [comment]
Code snippets aren't aligned with the grid on a mobile breakpoint.

code-snippet-mobile

**Comment by mmexvr** [comment]
Code snippets haven't been worked on yet

**Comment by niklasros** [comment]
The anchor link should on mobile be placed below the paragraph/introduction text.

anchor-link-position

**Comment by helloimela** [comment]
> The images along with the image captions are not aligned with the grid.
> 
> detailed-page-image-alignment

Image will be fixed on pull request #124

**Comment by niklasros** [comment]
Scenario: 

1. User scrolls on a detail page; _Foundation Spacing_.
2. User then navigates to another detail page, _Foundation Typography_, via the left navigation.
3. User then navigates back to the _Foundation Spacing_  via the left navigation and then ends up where it was when the user first left that specific page. 

It feels more natural if the detail page _always_ start at the top.

**Comment by helloimela** [comment]
> Scenario:
> 
> 1. User scrolls on a detail page; _Foundation Spacing_.
> 2. User then navigates to another detail page, _Foundation Typography_, via the left navigation.
> 3. User then navigates back to the _Foundation Spacing_  via the left navigation and then ends up where it was when the user first left that specific page.
> 
> It feels more natural if the detail page _always_ start at the top.

I added this to the backlog #128 
We should merge this Pull Request first @mmexvr so it does not grow too big.

**Review by mmexvr — APPROVED** [review]

**Pull request merged**